src/fparser/two/tests/utils/test_bracket_base.py

```diff
@@ -99,6 +99,29 @@ def test_cls():
             )
 
 
+def test_trailing_whitespace():
+    """Test that the BracketBase match method passes any trailing whitespace
+    within the brackets to the specified class.
+    """
+    for require in [False, True]:
```

Comment:
There's a nicer way of doing this using pytest:

    @pytest.mark.parametrize("require", [False, True])
    @pytest.mark.parametrize("lhs, rhs", [("(", ")"), (" ( ", " ) "), ...[]
    def test_trailing_whitespace(require, lhs, rhs):
        ...